features/check_standards/main_landmark.feature

```diff
@@ -21,10 +21,10 @@ Feature: Main landmark
       <div role="main">Main one</div>
       <div role="main">Main two</div>
       """
-    When I validate the landmark standards
+    When I validate the exactly one main landmark standards
     Then it fails with the message:
       """
-      A document must have exactly one main landmark. Found 2 elements with role="main".
+      Found 2 elements with role="main": /html/body/div[1] /html/body/div[2]
```

Comment:
To give you a bit of context... in this scenario the tool is complaining about offending elements in the page under test. XPaths are an important part of the failure message, because they enable the developer to identify which elements are the offenders (say, by using `x$(xpath)` in chrome dev tools).

We could make the xpaths cleverer (such as preferring ids over indexers) but then they would be more difficult and expensive to generate. We could use css selectors, but because we are targeting exactly one element per expression, css selectors actually turn out to be longer and harder to understand than xpaths (because they need things like nth-child pseudo selectors to distinguish between siblings).